contributing/workflow/pr_workflow.rst

```diff
@@ -248,6 +248,10 @@ section <doc_pr_workflow_rebase>` for instructions.
          to find the commit ID of the previous state that you would like to restore, and
          use it as argument of ``git reset --hard`` to go back to that state.
 
+If you had local changes before rebasing, the rebase rewrites your local git history.
+In that case, you will need to force push using ``git push --force origin master``
```

Comment:
This is wrong, unless you mean that the user should update their master branch with changes, we do not recommend making PR work on that branch, so should be worded differently

Reply:
That section already has language that assumes the user has their own commits to master, so it seems like there might be a larger structural issue here. I'm just going to back out of this rather than trying to work around that.

Reply:
That's because it's about updating your `master` branch, but it's vague, the general thing needs to be clarified that it's about preparing for a PR, and separate it out IMO